src/services/auth/signIn.js

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+
+export const signIn = async ({
+	loginData,
+	userData,
+	dispatch,
+	initialFormState,
+	toggleAuth,
+	rememberMe,
+	toggleLoader,
+}) => {
+	try {
+		toggleLoader();
+
+		const res = await axios.post("/api/auth/login", loginData);
```

Comment:
`{status, founder, encodedToken}` you can destruct this instead of using only res.

Reply:
Noted! Will update it.